File: 21/Status.java
import java.io.File;
import java.util.List;

import org.openqa.selenium.By;
import org.openqa.selenium.JavascriptExecutor;
import org.openqa.selenium.NoSuchElementException;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;


public class Status {

	public static void main(String[] args) throws InterruptedException {
		WebDriver dr = new ChromeDriver();
		
		File file = new File("src/status.html");
		String filePath = "file:///" + file.getAbsolutePath();
		System.out.printf("now accesss %s \n", filePath);
		
		dr.get(filePath);
		Thread.sleep(1000);	
		
		WebElement textField = dr.findElement(By.name("user"));
		System.out.println(textField.isEnabled());
		
//		直接用isEnabled方法去判断该button的话返回的会是true
//		这是因为button是使用css方法去disabled的，并不是真正的disable
//		这时候需要判断其class里是否有disabled这值来判断其是否处于disable状态
		System.out.println(dr.findElement(By.className("btn")).isEnabled());
		
//		隐藏掉textField
//		判断其是否显示
		((JavascriptExecutor)dr).executeScript("$(arguments[0]).hide()", textField);
		System.out.println(textField.isDisplayed());
		
//		使用click方法选择raido
		WebElement radio = dr.findElement(By.name("radio"));
		radio.click();
		System.out.println(radio.isSelected());
		
		try{
			dr.findElement(By.id("none"));
		} catch(NoSuchElementException e){
			System.out.println("element does not exist");
		}
		
		Thread.sleep(1000);
		System.out.println("browser will be close");
		dr.quit();	
	}

}

File: 21/status.py
# -*- coding: utf-8 -*- 
from selenium import webdriver
from time import sleep
import os
if 'HTTP_PROXY'in os.environ: del os.environ['HTTP_PROXY']

dr = webdriver.Chrome()
file_path = 'file:///' + os.path.abspath('status.html')
dr.get(file_path)

text_field = dr.find_element_by_name('user')
print text_field.is_enabled()

# 直接用enabled?方法去判断该button的话返回的会是true
# 这是因为button是使用css方法去disabled的，并不是真正的disable
# 这时候需要判断其class里是否有disabled这值来判断其是否处于disable状态
print dr.find_element_by_class_name('btn').is_enabled()

# 隐藏掉text_field
# 判断其是否显示
dr.execute_script('$(arguments[0]).hide()', text_field)
print text_field.is_displayed()

# 使用click方法选择raido
radio = dr.find_element_by_name('radio')
radio.click()
print radio.is_selected()

# 判断元素是否存在
try:
	dr.find_element_by_id('none')
except: 
	print 'element does not exist'

dr.quit()

